.github/workflows/code-quality.yml

```
@@ -26,7 +26,7 @@ jobs:
           sudo apt-get update
           sudo apt-get install -y shellcheck shfmt ripgrep dnsmasq
 
-      - name: Run code quality checks
+        run: sh tools/code-quality.sh
```

actionlint (1.7.12)

[error] key "run" is duplicated in element of "steps" section. previously defined at line:25,col:9

(syntax-check)